tests/index.test.ts

```diff
@@ -152,6 +163,55 @@ describe('postcss-theme-fold', () => {
       )
     })
 
+    test('should not generate css-rules if all variables are missing in theme', async () => {
+      await run(
+        `.Textinput-Box {
+          border-color: var(--color-3);
+          background-color: var(--color-4);
+        }`,
+        `
+          .Theme_color_b .Textinput-Box {
+            border-color: #fff;
+            background-color: #ccc;
+          }
+        `,
+      )
+    })
+
+    test('should not touch variables, which are missing in all themes', async () => {
+      await run(
+        `.Textinput-Box {
+          background: black var(--color-nonexistent) no-repeat;
+          background-color: var(--color-bg-nonexistent);
+        }`,
+        `.Textinput-Box {
+          background: black var(--color-nonexistent) no-repeat;
+          background-color: var(--color-bg-nonexistent);
+        }`,
+      )
+    })
```

Comment:
Harmless variables (?)
Can possible broke build, but who will try to use variables without values?..

Reply:
Hm, this test case already exists:

https://github.com/yarastqt/postcss-theme-fold/blob/7b942c29ffec6431fb24b5c3948c13cbfba6816e/tests/index.test.ts#L109-L114